java/test/jmri/jmrix/AbstractThrottleTest.java

```diff
@@ -766,7 +767,7 @@ public void testThrottleDispose() {
      */
     @Test
     public void testGetSpeedIncrement() {
-        float expResult = 0.0F;
+        float expResult = 1.0F;
```

Comment:
This change implies the default increment value changed from no speed increment ( i.e. never increase the speed ) to “only have one increment”.  In other words, immediately go from 0 speed to full speed.  I am not sure this is right.

Reply:
Yeah, this is a bit of a degenerate case when the speed step mode for the throttle isn't set. I'm not sure what the right answer is, or how the throttle UI handles this; do you have any suggestions?

Reply:
Remember that this is testing the abstract throttle.  

I think a safer default, in the case that the throttle doesn't set it itself, is for 0.0.  Setting this to 0 means that the UI won't change when a developer is testing it in the the case when a derived class implementation fails to update the value.

Reply:
Now that the speed increment is derived from the speed step mode, there's no chance that the developer will forget to update it.

I've updated the default speed step mode to have one step with an increment of 0, to emulate the previous behavior.